opal/mca/reachable/netlink/configure.m4

```diff
@@ -71,74 +72,37 @@ AC_DEFUN([OPAL_REACHABLE_NETLINK_CHECK_LIBNL3],[
-       AS_IF([test -d "/usr/include/libnl3"],
-             [AC_MSG_RESULT([present])
-              AC_CHECK_HEADER(
-                [netlink/version.h],
-                [AC_COMPILE_IFELSE(
-                    [AC_LANG_PROGRAM([[
-#include <netlink/netlink.h>
-#include <netlink/version.h>
-#ifndef LIBNL_VER_MAJ
-#error "LIBNL_VER_MAJ not defined!"
-#endif
-/* to the best of our knowledge, version.h only exists in libnl3 */
-#if LIBNL_VER_MAJ < 3
-#error "LIBNL_VER_MAJ < 3, this is very unusual"
-#endif
-                        ]],[[/* empty body */]])],
-                    [HAVE_LIBNL3=1],    dnl our program compiled
-                    [HAVE_LIBNL3=0])],  dnl our program failed to compile
-                [HAVE_LIBNL3=0],  dnl AC_CHECK_HEADER failed
-                [#include <netlink/netlink.h>
-                ])],
-             [AC_MSG_RESULT([missing])
-              HAVE_LIBNL3=0])  dnl "/usr/include/libnl3" does not exist
-
-       # nl_recvmsgs_report is a symbol that is only present in v3
-       AS_IF([test "$HAVE_LIBNL3" -eq 1],
-             [AC_SEARCH_LIBS([nl_recvmsgs_report], [nl-3],
-                             [# We also need libnl-route-3
-                              AC_SEARCH_LIBS([nl_rtgen_request], [nl-route-3],
-                                             [$2="-lnl-3 -lnl-route-3"
-                                              HAVE_LIBNL3=1],
-                                             [HAVE_LIBNL3=0])],
-                             [HAVE_LIBNL3=0])])
-
-       AS_IF([test "$HAVE_LIBNL3" -eq 1],
-             [AC_MSG_NOTICE([using libnl-3])],
-             [# restore $1 since we are falling back to libnl (v1)
-              $1=""
-              AC_SEARCH_LIBS([nl_connect], [nl],
-                             [$2="-lnl"],
-                             [AC_MSG_WARN([Cannot find libnl-3 nor libnl])
-                              AS_IF([test "$3" = "1"],
-                                    [AC_MSG_ERROR([Cannot continue])])
-                             ])
-              AC_MSG_NOTICE([using libnl (v1)])])
-
-       # libnl_utils.h does not include configure-generated config.h,
-       # so it may not see the HAVE_LIBNL3 #define.  Hence, we set
-       # HAVE_LIBNL3 as both a C preprocessor macro (in case some
-       # other file includes config.h before libnl_utils.h) and a
-       # Makefile macro (so that the app can set HAVE_LIBNL3 via
-       # CPPFLAGS).  Also, this macro may be used in multiple
-       # different libraries; setting HAVE_LIBNL3 both ways lets the
-       # application choose which way to set it.
-       AC_SUBST([HAVE_LIBNL3])
-       AC_DEFINE_UNQUOTED([HAVE_LIBNL3],[$HAVE_LIBNL3],
-                          [set to 1 if should use libnl v3, set to 0 for libnl v11])
-
-       LIBS=$save_LIBS
-       AS_UNSET([save_LIBS])
-       CPPFLAGS=$save_CPPFLAGS
-       AS_UNSET([save_CPPFLAGS])
+	# Default to a numeric value (this value gets AC_DEFINEd)
+	HAVE_LIBNL3=0
```

Comment:
It looks like this AC_DEFINE is specific to the reachable/netlink component.  It should be renamed to follow the prefix rule.